tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_create_new_alarm.py

```diff
@@ -34,19 +34,19 @@ def test_clock_create_new_alarm(self):
         alarm_label_text = "test4321"
 
         # get the number of alarms set, before adding the new alarm
```

Comment:
We can apply the [boy scout rule](http://www.informit.com/articles/article.aspx?p=1235624&seqNum=6) here. This comment has no value, as getting the number of things before doing an action is a common testing pattern.